jcl/src/java.base/share/classes/module-info.java.extra

```diff
@@ -30,6 +30,7 @@ exports com.ibm.sharedclasses.spi to openj9.sharedclasses, java.management, java
 exports com.ibm.oti.vm to java.management, jdk.attach, jdk.management, openj9.jvm, openj9.sharedclasses;
 exports com.ibm.oti.util to java.management, jdk.attach, jdk.jcmd, jdk.management, openj9.sharedclasses;
 exports com.ibm.tools.attach.target to jdk.attach, jdk.jcmd, jdk.management;
+exports openj9.management to java.management;
```

Comment:
I suggest `openj9.management.internal` is a better place for the classes below.

Reply:
Good idea. Renamed accordingly.